docs/modules/user_guide/tree.rst

```diff
@@ -0,0 +1,116 @@
+.. _tree:
+
+===========================
+Unsupervised Decision Trees
+===========================
+
+.. currentmodule:: sklearn.tree
+
+In unsupervised learning, the goal is to identify patterns 
+or structure in data without using labeled examples. Clustering is a common 
+unsupervised learning technique that groups similar examples together 
+based on their features. In clustering settings, there may be no natural 
+notion of “true” class-labels, thus the efficacy of the clustering scheme is 
+often measured based on distance based metrics such as :func:`sklearn.metrics.adjusted_rand_score`.
```

Comment:
Can you add a refernece/link to the scikit-learn documentation regarding clustering metrics? 

https://scikit-learn.org/stable/modules/clustering.html#clustering-performance-evaluation